_posts/en/newsletters/2019-07-24-newsletter.md

```diff
@@ -0,0 +1,149 @@
+---
+title: 'Bitcoin Optech Newsletter #56'
+permalink: /en/newsletters/2019/07/24/
+name: 2019-07-24-newsletter
+slug: 2019-07-24-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes progress on signet and an idea for just-in-time
+routing on LN.  Also included are our regular sections about bech32
+sending support and notable changes to popular Bitcoin infrastructure
+projects.
+
+{% comment %}<!-- include references.md below the fold but above any Jekyll/Liquid variables-->{% endcomment %}
+{% include references.md %}
+
+## Action items
+
+- **Help test Bitcoin Core 0.18.1 release candidates:** this upcoming
+  maintenance release fixes several bugs that affected certain RPC
+  commands and caused unnecessarily high CPU use in certain cases.
+  Production users are encouraged to test the current [release
+  candidate][core rc] to ensure that it operates as expected.
+
+## News
+
+- **Progress on signet:** signet is a testnet alternative where valid
+  blocks are signed by a trusted authority.  The authority generally produces
+  a regular series of blocks but may occasionally produce forks leading
+  to block chain reorganizations.  This avoids commonly-encountered
+  problems on testnet such as too-fast block production, too-slow block
+  production, and reorganizations involving thousands of blocks.
+  Subsequent to our earlier report on signet in [Newsletter #37], its
+  author has created an implementation, documented it on a [wiki
+  page][signet wiki], opened a [PR][Bitcoin Core #16411] proposing to
+  add support for signet to Bitcoin Core, and posted a [draft
+  BIP][signet bip post] to the Bitcoin-Dev mailing list.
+  The proposed implementation also makes it easy for teams to create
+  their own independent signets for specialized group testing, e.g.
+  signet author Kalle Alm reports that "someone is already working on a
```

Comment:
Perhaps re-introduce Kalle Alm in L34 above when the author is first mentioned, and here remove "signet author" in favour of just the name.